images/pytorch-notebook/README.md

```diff
@@ -1,4 +1,4 @@
-# Jupyter Notebook Deep Learning Stack
+# Jupyter Notebook PyTorch Deep Learning Stack
```

Comment:
We have the same naming for PyTorch and TensorFlow images, I decided to change them slightly.